src/components/common/Avatar.vue

```diff
@@ -4,23 +4,23 @@
 </template>
 
 <script>
-import Avatars from '@dicebear/avatars'
-import sprites from '@dicebear/avatars-jdenticon-sprites'
+import Avatars from "@dicebear/avatars"
+import sprites from "@dicebear/avatars-jdenticon-sprites"
 
 export default {
   name: `avatar`,
   props: {
     address: {
       type: String,
-      required: true
-    }
+      required: true,
+    },
   },
   computed: {
     svg() {
       const options = {}
       const avatars = new Avatars(sprites, options)
-      return avatars.create(this.address)
-    }
-  }
+      return avatars.create(this.address, options)
```

Comment:
this is needed? https://github.com/DiceBear/avatars/tree/master/packages/avatars

Reply:
that's why I should open an issue to these guys 😅 It is in the PR description. Documentation outdated at least for jidenticons

Reply:
Sure? Actually the old version should work too. If not, can you describe the problem? 😅

Reply:
Hi Florian! Actually this was me messing things up 😅 Don't worry, your new version works great too with the current documentation.

I just needed to update my packages, it was as simple as that.

Sorry for the confusion here 🙏 
And thank you for being so attentive!